src/commands/role.ts

```diff
@@ -77,18 +78,36 @@ class RoleCMD extends Command {
     private async listRoles(params: Params): Promise<void> {
         const { guild, message, bot } = params;
         const botMember: GuildMember = await guild.members.fetch(bot.id);
-        const roleList = this.getAvailableRoles(guild, botMember);
+        const roleList = Object.keys(
+            this.getAvailableRoles(guild, botMember)
+        ).sort();
```

Comment:
Out of curiosity, Is there any reason to `sort` the list? Does this improve the current implementation?

Reply:
Feel free to dissmiss the commit, but the unsorted version will look like:
```
buddies           docker
python            django
seo               java
typescript        digital-marketing
node              c-cpp
game-developer    react-native
security          nextjs
ethical-hacking   vue
graphic-design    flutter
php               product-design
javascript        react
kubernetes        flask
ui-ux             dart
blogger
```
Which makes it slightly hard to search for something. If you sort it, it will look like:
```
blogger             kubernetes
buddies             nextjs
c-cpp               node
dart                php
digital-marketing   product-design
django              python
docker              react
ethical-hacking     react-native
flask               security
flutter             seo
game-developer      typescript
graphic-design      ui-ux
java                vue
javascript
```
It is merely cosmetic, but in the first one, `react` and `react-native` are quite apart and it is easy to not notice there are two, whilst in the later you have `react` and then `react-native`.

Reply:
Amazing, this improves user experience. Great work. 👍